README.md

```diff
@@ -2,6 +2,8 @@
 <img src="docs/logos/helics-logo-long-primary-black.svg" width="700">
 </p>
 
+A multi-language, cross-platform framework that enables different simulators to easily exchange data and stay synchronized in time. Scalable from two simulators on a laptop to 10,000+ running on high-performance computers, the cloud, or a mix of all of these.
```

Comment:
"high-performance computers" -> "HPC" or "supercomputers" maybe?
"a mix of all of these" -> "a mix of these"?

Reply:
I think changing to Supercomputers would be good

Reply:
```suggestion
A multi-language, cross-platform framework that enables different simulators to easily exchange data and stay synchronized in time. Scalable from two simulators on a laptop to 10,000+ running on supercomputers, the cloud, or a mix of these platforms.
```